packages/types/src/global-settings.ts

```diff
@@ -50,6 +50,7 @@ export const globalSettingsSchema = z.object({
 	autoCondenseContext: z.boolean().optional(),
 	autoCondenseContextPercent: z.number().optional(),
 	maxConcurrentFileReads: z.number().optional(),
+	blockVeryLargeReads: z.boolean().optional(), // kilocode_change
```

Comment:
if we make this the default behaviour, wouldnt allowVeryLargeReads which is false by default make more sense?